examples/test.py

```diff
@@ -0,0 +1,207 @@
+from dataclasses import asdict, dataclass, field
+from enum import Enum
+from typing import Any, List
+
+from hydra import main
+from hydra.core.config_store import ConfigStore
+from omegaconf import MISSING, DictConfig
+
+
+# @dataclass
```

Comment:
I am leaving the test.py and test.yaml files for now just because I am showing here how I envision parametrization of Jasper. 

So please do not discuss the details of this or that - simply those two files will be gone soon after we'll reach alignment.

Reply:
sounds good